internal/resources/slo/resource_slo.go

```diff
@@ -552,6 +568,20 @@ func packAlerting(tfAlerting map[string]interface{}) slo.SloV00Alerting {
 		SlowBurn:    &tfSlowBurn,
 	}
 
+	// All options in advanced options will be optional
+	// Adding a second feature will need to make a better way of checking what is there
+	if failures := tfAlerting["advanced_options"]; failures != nil {
+		lf := failures.([]interface{})
```

Comment:
```suggestion
	if advancedOptions, ok := tfAlerting["advanced_options"].([]interface{}); ok {
```

it seems that the blocks above this all follow the same pattern with type assertion on the "if" line. Can we not follow that pattern here? Note: the suggestion is using `[]interface{}` not `[]int` (type depends on whether you take suggestion above)